### PR TITLE
Adding Text sprites

### PR DESCRIPTION
**Pull request by Licini**
#32 
example: `scripts/v120_text.py`
image

### CODE REVIEW

**Comment by Licini** [comment]
@brgcode @GeneKao This one is ready. Let me know what do you guys think

**Comment by GeneKao** [comment]
It works on my computer (macOS Big Sur 11.2.2) without any problem. It's nice! 😉

File: src/compas_view2/objects/textobject.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            'elements': make_index_buffer([0]),
+            'text_texture': self.make_text_texture(),
+            'n': 1
+        }
+
+    def make_text_texture(self):
+        # change the filename if necessary
+        face = ft.Face(os.path.join(DATA, "FreeSans.ttf"))
+        # the size is specified in 1/64 pixel
+        face.set_char_size(48*64)
```
Comment (brgcode) [inline src/compas_view2/objects/textobject.py:38]: if this is the font size, should it not be a parameter?

File: src/compas_view2/views/view120.py [hunk 2]
```diff
@@ -26,6 +28,13 @@ def init(self):
...
 
+        self.shader_text = Shader(name='120/text')
+        self.shader_text.bind()
+        self.shader_text.uniform4x4("projection", self.camera.projection(self.app.width, self.app.height))
+        self.shader_text.uniform4x4("viewworld", self.camera.viewworld())
+        self.shader_text.uniform1f("opacity", self.opacity)
+        self.shader_text.release()
+
     def resize(self, w, h):
         self.shader.bind()
```
Comment (brgcode) [inline src/compas_view2/views/view120.py:39]: should we rename the default shader to `shader_model` or something like that as well?

File: src/compas_view2/objects/textobject.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
+from .object import Object
+from compas_view2 import DATA
```
[the remark this answers is not in the document]
Comment (Licini) [reply src/compas_view2/objects/textobject.py:2]: sure, how it should be called and where? `./fonts`, `./data/fonts` or `./src/compas_view2/shapes/fonts` ?

File: src/compas_view2/views/view120.py (hunk 2 above)
Comment (Licini) [reply src/compas_view2/views/view120.py:39]: yep sounds good will do it

File: src/compas_view2/objects/textobject.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (brgcode) [reply src/compas_view2/objects/textobject.py:2]: `./fonts`

**Comment by Licini** [comment]
@brgcode Ok adjustments on comments are made,  @GeneKao The bug you mentioned is now fixed

**Pull request merged**